src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -587,16 +587,16 @@ public int getPhysicalProcessorCount() {
     @Override
     public String toJSON() {
         StringBuilder sb = new StringBuilder("{");

```

Comment:
There's got to be a JSON parser where you can turn a Dictionary into JSON which takes care of all of this ...